sqlglot/dialects/nuodb.py

```diff
@@ -1,9 +1,35 @@
 from __future__ import annotations
 
-from sqlglot import exp, generator, parser, tokens
-from sqlglot.dialects.dialect import Dialect
+from sqlglot import exp, generator, parser, tokens, transforms
+from sqlglot.dialects.dialect import Dialect, no_properties_sql
 from sqlglot.tokens import Tokenizer, TokenType
 
+def _auto_increment_to_generated_by_default(expression: exp.Expression) -> exp.Expression:
+    auto = expression.find(exp.AutoIncrementColumnConstraint)
+    if auto:
+        expression = expression.copy()
+        expression.args["constraints"].remove(auto.parent)
+        kind = expression.args["kind"]
+        if auto is not None:
```

Comment:
isn't this redundant, as the previous conditional statement checks if `auto` is initialized already?